devsds/lib/opensds.sh

```diff
@@ -23,8 +23,8 @@ set +o xtrace
 soda:opensds:configuration(){
 
 # Copy api spec file to configuration path
-SODA_API_DIR=${SODA_DIR}/../../api
-cp $SODA_API_DIR/openapi-spec/swagger.yaml $SODA_CONFIG_DIR
+SODA_API_DIR=/opt/opensds-hotpot-linux-amd64
```

Comment:
Just trying to understand, with only **devsds** build, who creates this /opt/opensds-hotpot-linux-amd64 dir?

Reply:
some more doubts.... why we need this path?
devsds is for developer version and in many case...they will clone at /github/sodafoundation/xxx. How to handle this case?